### DISCUSSION TITLE
Forcing auth as ntlmv2

### DISCUSSION THREAD

**Discussion by Luffy610** [genotrance/px#216] [Q&A] [answered]
We have a usecase to default our auth to NTLMv2 as default for security purposes is it possible to force it to authenticate with ntlmv2?

**Comment by genotrance** [comment, marked as answer]
Looks like this needs to be configured in [Windows](https://learn.microsoft.com/en-us/troubleshoot/windows-client/windows-security/enable-ntlm-2-authentication). libcurl simply has a flag for [NTLM](https://curl.se/libcurl/c/CURLOPT_HTTPAUTH.html), nothing about versions.
